src/components/Input/styles.ts

```diff
@@ -1,15 +1,16 @@
-import colors from '@src/constants/colors';
 import { StyleSheet } from 'react-native';
 
+import colors from '@constants/colors';
+
 const styles = StyleSheet.create({
   inputContainer: {
     flexDirection: 'row',
     width: '100%',
-    backgroundColor: '#f1f4ff',
+    backgroundColor: colors.background,
     borderRadius: 10,
     padding: 10,
     gap: 10,
-    borderWidth: 2,
+    borderWidth: 1,
     borderColor: 'rgba(0, 0, 0, 0)',
```

Comment:
color constant

Reply:
All the above colors are replaced with color constants in the latest commit